client/my-sites/invites/index.js

```diff
@@ -6,11 +6,12 @@ import page from 'page';
 /**
  * Internal dependencies
  */
-import { acceptInvite } from './controller';
```

Comment:
Personally I find it more beautiful :relaxed: if we do `{ acceptInvite, saveSubscriptionActivationKey }` instead of importing `controller`